conda/build-environment.yaml

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Build environment.
+name: tvm-build
+
+channels:
+  - anaconda
+  - conda-forge
+
+dependencies:
```

Comment:
do we want to pin more/all of these dependencies?

Reply:
I only pined llvm version since that is most relevant, as a package we want to be flexible to versions and most of the core deps are relatively stable. (we are not testing against e2e like tf/pytorch which will need pinning)

Reply:
ok thanks